plenum/test/bls/test_bls_bft_replica.py

```diff
@@ -339,6 +372,81 @@ def test_process_order(bls_bft_replicas, pre_prepare_no_bls, quorums):
                               pre_prepare_no_bls)
 
 
+# ------ MULTIPLE MULTI_SIGS ------
+
+
+def test_update_pre_prepare_after_ordered_with_multiple_sigs(bls_bft_replicas, state_root, fake_multi_sig, ledger_id,
+                                                             create_audit_txn_with_multiple_ledgers):
+    for bls_bft_replica in bls_bft_replicas:
+        bls_bft_replica._bls_latest_multi_sig = fake_multi_sig
+        bls_bft_replica._all_bls_latest_multi_sigs = [fake_multi_sig]
+
+    params = create_pre_prepare_params(state_root, ledger_id=ledger_id)
+
+    params_initial = copy(params)
+    for bls_bft_replica in bls_bft_replicas:
+        params_res = bls_bft_replica.update_pre_prepare(copy(params), ledger_id)
+        assert params_res != params_initial
+        assert len(params_res) - len(params_initial) == 2
+
+
+def test_update_commit_with_multiple_sigs(bls_bft_replicas, fake_pre_prepare_with_bls,
+                                          create_audit_txn_with_multiple_ledgers):
+    params = create_commit_params(0, 0)
+    params_initial = copy(params)
+    for bls_bft_replica in bls_bft_replicas:
+        params_res = bls_bft_replica.update_commit(copy(params), fake_pre_prepare_with_bls)
+        assert params_res != params_initial
+        assert len(params_res) - len(params_initial) == 2
+
+
+def test_validate_pre_prepare_multiple_correct_multi_sigs(bls_bft_replicas, pre_prepare_with_bls_multi,
+                                                          create_audit_txn_with_multiple_ledgers):
+    for sender_bls_bft_replica in bls_bft_replicas:
+        for verifier_bls_bft_replica in bls_bft_replicas:
+            assert not verifier_bls_bft_replica.validate_pre_prepare(pre_prepare_with_bls_multi,
+                                                                     sender_bls_bft_replica.node_id)
+
+
+def test_validate_commit_incorrect_sig_with_multiple_sigs(bls_bft_replicas, pre_prepare_with_bls,
+                                                          create_audit_txn_with_multiple_ledgers):
+    key = (0, 0)
+    for sender_bls_bft in bls_bft_replicas:
+        fake_sig = base58.b58encode(b"somefakesignaturesomefakesignaturesomefakesignature").decode("utf-8")
+        commit = create_commit_with_bls_sigs(key, fake_sig)
```

Comment:
Do we have multiple sigs here? 
Please add more tests for validation: 
- when we really have more than 1 sig
- when some of this sigs are valid and some are not
- when there is multi-sig field, but it's empty